docs/examples/Parameters/Complex_Parameters.ipynb

```diff
@@ -151,7 +151,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The proper validator should accept complex numbers and should reject anything of the wrong shape. Note that we get to decide whether we want to accept \"non-strictly complex\" data."
+    "The proper validator should accept complex numbers and should reject anything of the wrong shape. Note, that we get to decide whether we want to accept \"non-strictly complex\" data."
```

Comment:
Are you sure about this comma?

Reply:
No, after googling a bit, it does not look like a correct choice. I generally use a comma after an introductory phrase such as however, therefore, and other such adverbs. That logic leaked into placing the comma after `Note`.